packages/liveblocks-react/src/umbrella-store.ts

```diff
@@ -243,21 +279,26 @@ export type UmbrellaStoreState<M extends BaseMetadata> = {
 };
 
 export class UmbrellaStore<M extends BaseMetadata> {
+  private _client?: OpaqueClient;
```

Comment:
Can we make `client` required? Is there a particular reason why it is not?

Reply:
Only didn't do so because we would have to create a client in all unit tests, which won't be used by most of them at the moment.